net/xl2tpd/patches/110-makefile_dont_build_pfc.patch

```diff
@@ -1,43 +0,0 @@
---- a/Makefile
-+++ b/Makefile
-@@ -113,10 +113,10 @@ BINDIR?=$(DESTDIR)${PREFIX}/bin
- MANDIR?=$(DESTDIR)${PREFIX}/share/man
- 
- 
--all: $(EXEC) pfc $(CONTROL_EXEC)
-+all: $(EXEC) $(CONTROL_EXEC)
- 
- clean:
--	rm -f $(OBJS) $(EXEC) pfc.o pfc $(CONTROL_EXEC)
-+	rm -f $(OBJS) $(EXEC) $(CONTROL_EXEC)
- 
- $(EXEC): $(OBJS) $(HDRS)
- 	$(CC) $(LDFLAGS) -o $@ $(OBJS) $(LDLIBS)
-@@ -124,14 +124,10 @@ $(EXEC): $(OBJS) $(HDRS)
- $(CONTROL_EXEC): $(CONTROL_SRCS)
- 	$(CC) $(CFLAGS) $(LDFLAGS) $(CONTROL_SRCS) -o $@
- 
--pfc:
--	$(CC) $(CFLAGS) -c contrib/pfc.c
--	$(CC) $(LDFLAGS) -o pfc pfc.o -lpcap $(LDLIBS)
```

Comment:
pfc depends on libpcap and the build can potentially fail if it's not present, so I think this patch is still needed

Reply:
ppp has PKG_BUILD_DEPENDS:=libpcap do you think xl2tpd will be used without ppp ?

Reply:
I missed this indirect dependency, but I think direct deps should be explicitly specified.  You dont build pfc or if you build it you should tell the build system and the reader what it requires